core/src/main/scala/kafka/server/AbstractFetcherThread.scala

```diff
@@ -145,10 +145,9 @@ abstract class AbstractFetcherThread(name: String,
                 case Errors.NONE =>
                   try {
                     val messages = partitionData.toByteBufferMessageSet
+                    val validBytes = messages.validBytes
```

Comment:
Later, `validBytes` has been used twice. So I assign the val to avoid calling `messages.validBytes` method twice. 
I put the assignment here instead of just above the statement of `if (validBytes > 0) {` because I think it is better to place all `messages` related statements as close as possible. It would be easier for future developers if they need to revise `messages`.

Let me know what is your preference. Thanks.